horizomer/utils/tree.py

```diff
@@ -15,6 +15,59 @@
 from skbio import TreeNode
 
 
+def support(node):
+    """Get support value of a node.
+
+    Parameters
+    ----------
+    node : skbio.TreeNode
+        node to get support value of
+
+    Returns
+    -------
+    float or None
+        support value of the node, or None if not available
+
+    Notes
+    -----
+    A "support value" is defined as the numeric form of a whole node label
+    without ":", or the part preceding the first ":" in the node label.
+    - For examples: "(a,b)1.0", "(a,b)1.0:2.5", and "(a,b)'1.0:species_A'". In
+    these cases the support values are all 1.0.
+    - For examples: "(a,b):1.0" and "(a,b)species_A". In these cases there are
+    no support values.
+    """
+    try:
```

Comment:
that seems to be an ugly hack. Is TreeNode not able to parse support values in the appropriate way? I know it is kind of limited. Maybe you want to parse the tree via ete and than convert the object to TreeNode, which can have additional parameters on each node, like ```node.support = 1.0```

Reply:
As far as I understand, TreeNode cannot handle support values. A support value is but treated as a node label. If we use ETE3 basically we are replacing scikit-bio in all tree operations...

Reply:
yikes!  That is pretty stemming from some the newick parser - it can't handle extra metadata very well.  Probably should raise an issue on this on skbio.

Reply:
Oh man it does seem like handling node-linked metadata is kind of a must for TreeNode. It doesn't seem reasonable to make the TreeNode logic itself limited by the dumpster fire that is the Newick un-format. 

@wasade any thoughts on this?

Reply:
Hi @tanaes As far as I understand, the standard Newick format has no place for "node support value" specifically. When you load a tree with bootstraps in FigTree, it asks if the values are "label". In some Newick derivatives (such as BEAST trees), metadata fields are included in `[]`. I think it is ideal though not absolutely essential to add those parsers.